.pre-commit-config.yaml

```
@@ -0,0 +1,16 @@
+repos:
+  -   repo: https://github.com/timothycrosley/isort
```

The repo URL changed:

```suggestion
  -   repo: https://github.com/PyCQA/isort
```